src/microbe_stage/systems/MicrobeVisualsSystem.cs

```diff
@@ -347,29 +354,6 @@ private void CreateOrganelleVisuals(Node3D parentNode, ref OrganelleContainer or
 
                 organelleContainer.CreatedOrganelleVisuals.Add(placedOrganelle, visualsInstance);
             }
-
-            // Visuals already exist
-            var graphics = placedOrganelle.OrganelleGraphics;
```

Comment:
Is there a reason why you removed this code? If I'm seeing things correctly then you haven't replaced this with anything meaning the organelle visuals are now gone.

Reply:
The `graphics` variable was only needed to assign shader variables to it; the graphics themselves are created before, so this change doesn't really remove the visuals.

Anyway, after some thought, I decided to revert removing this code, so that tint might still be added to organelles if needed.

Reply:
But if tint was removed, doesn't that make all species look very similar? As the colour was kind of the primary way (well besides size and some really distinct organelles) to quickly identify species.

Reply:
The tint in this function is only applied to organelles (after all, it's called `CreateOrganelleVisuals`), while mebranes' tint is left as is